test/runtests.jl

```diff
@@ -42,6 +42,12 @@ a1 = Graph(adjmx1)
 a2 = DiGraph(adjmx2)
 
 tests = [
+    "operators",
+    "community/core-periphery",
+    "community/detection",
+    "community/modularity",
+    "community/clustering",
+    "randgraphs",
     "graphdigraph",
```

Comment:
Gah. Sorry - one last change. Can you please preserve the oder of runtests? That is, graphdigraph should be first, operators should be near the top, and community/ stuff should be towards the end. I know it's useful for testing pre-merge, but we should switch it back before committing.